assignment2/inheritance.js

```diff
@@ -0,0 +1,17 @@
+function Employee (name) {
+    this.name = name;
+    
+                          }
+
```

Comment:
If indentation would have been a person, then this PR would have killed it! RIP Indentation!!